{{cookiecutter.project_slug}}/.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,65 @@
+# This workflow will install Python dependencies, run tests and lint with multiple version of Python
+name: matchms
+
+on:
```

Comment:
The deployment should also be done on a release not for each push to master. Again see https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries for an example.

Reply:
already fixed.